src/index.html

```diff
@@ -17,7 +17,7 @@
       }
     </style>
   </head>
-  <body>
+  <body style="background-color: #F1F1F1;">
```

Comment:
On donate frontend the background is #f4f4f4 ( https://github.com/thebiggive/donate-frontend/blob/5bc048de570bd36e0b857f70ad8c7ba380a573e0/src/assets/scss/_variables.scss#LL1C1-L1C1 ). Is it worth setting this the same?

Reply:
This is just a scratch pad for previewing. No need for it to be the same

Reply:
OK - the only reason I thought it might be worth it being the same was there are some components that are meant to appear non-rectangular, using the hack of setting their background to the same as the containing element background.